site/src/components/context/UserContext.ts

```diff
@@ -0,0 +1,16 @@
+import { createContext, Dispatch, SetStateAction } from "react";
+import { SerializedUser } from "../../lib/model/user.model";
+
```

Comment:
I think we should move the `context` folder up since it isn't exactly a component.
Structure should look like this

```
site
  - context
  - hooks
  - components
    - Button
    - Modal
  - pages
```

instead of 

```
site
  - components
     - context
     - Modal
  - pages     
```

Reply:
I have no objections to that, done this in https://github.com/blockprotocol/blockprotocol/pull/84/commits/eac5c0b3d7f9cb9ac1a85570be221ac2bfc7724a